### ISSUE TITLE
Color contrast does not meet minimum requirement: Bulkrax Field Mappings

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3077]
vpat.hykudemo.org
Version 7.1

Descriptive summary
Color contrast for the language and login drop-down menus is not clear enough:
<img width="1152" height="970" alt="Image" src="https://github.com/user-attachments/assets/8cc49f52-919a-4f36-b2ed-5ae4d38790f0" />

Rationale
Accessibility issue remediation

Text and background colors that conflict or have too little contrast between each other can cause readability issues for users who are color blind or have low vision (vision loss equivalent to approx. 20/40 vision).

To meet the WCAG contrast requirement, you should have a contrast ratio of 4.5:1 for normal text, and 3:1 for large text (18pt and above, or 14pt bold).

Expected behavior
Color contrast should be at least 4.5: 1

Actual behavior
Color contrast is 3.01:1

Steps to reproduce the behavior

Open the page
Use the SiteImprove browser extension

Related: https://github.com/samvera/hyku/issues/3075
